bdl-vscode/src/diagnostics.ts

```diff
@@ -149,16 +149,19 @@ function checkDuplicatedAttributeNames(
   }
 }
 
-function checkDuplicatedEnumAndUnionItems(
+function checkDuplicatedNodeItems(
```

Comment:
how about `checkDuplicatedItemNames``

Reply:
we already check for union items, are there more things to check?

Reply:
ah okay